--- /tf-dev-env/vrouter/Makefile.orig   2020-11-12 04:26:23.058686300 +0000
+++ /tf-dev-env/vrouter/Makefile        2020-11-12 04:31:39.668686300 +0000  
@@ -82,6 +82,7 @@
        ccflags-y += -I$(src)/include -I$(SANDESH_HEADER_PATH)/sandesh/gen-c 
        ccflags-y += -I$(SANDESH_EXTRA_HEADER_PATH)
        ccflags-y += -I$(SANDESH_EXTRA_HEADER_PATH)/sandesh/library/c        
+       ccflags-y += -Wno-maybe-uninitialized
        ccflags-y += -g -Wall
     ccflags-y += -g -Werror
     ccflags-$(CONFIG_RETPOLINE) += -DRETPOLINE
