--- tf-dev-env/vrouter/Makefile.orig	2020-11-12 22:44:28.730000000 +0000
+++ tf-dev-env/vrouter/Makefile	2020-11-12 22:45:20.370000000 +0000
@@ -82,6 +82,7 @@
 	ccflags-y += -I$(src)/include -I$(SANDESH_HEADER_PATH)/sandesh/gen-c
 	ccflags-y += -I$(SANDESH_EXTRA_HEADER_PATH)
 	ccflags-y += -I$(SANDESH_EXTRA_HEADER_PATH)/sandesh/library/c
+	ccflags-y += -Wno-maybe-uninitialized
 	ccflags-y += -g -Wall
     ccflags-y += -g -Werror
     ccflags-$(CONFIG_RETPOLINE) += -DRETPOLINE
